config/settings/test.yml

```diff
@@ -1,2 +1,16 @@
 workflow:
   url: 'http://workflow.example.com/workflow'
+
+sdr:
+  local_workspace_root: /dor/workspace
+  abbyy:
+    # Windows-style paths for the ABBYY machine, referenced in XML tickets
+    remote_output_path: E:\AbbyyShare\sdr-ocr-qa\OUTPUT
+    remote_result_path: E:\AbbyyShare\sdr-ocr-qa\RESULTXML
+    # Unix-style paths for the common-accessioning VM, used to manage files
+    local_ticket_path: /tmp/ABBYY/INPUT
+    local_result_path: /tmp/ABBYY/RESULTXML
+    local_output_path: /tmp/ABBYY/OUTPUT
+    local_exception_path: /tmp/ABBYY/EXCEPTIONS
+
```

Comment:
do we really need to override all of these values for the tests?  they are set in `config/settings.yml` to various tmp values

Reply:
Yeah, you're right they aren't needed. I removed them.